static/app/api.tsx

```diff
@@ -581,3 +585,60 @@ export class Client {
     );
   }
 }
+
+function generateOrganizationBaseUrl(organizationUrl: string) {
+  return `${organizationUrl}${DEFAULT_BASE_URL}`;
+}
+
+type APIRouteType = 'organization-events';
+
+type LegacyRoute = string;
+type CustomerDomainRoute = string;
+type RouteTuple = [LegacyRoute, CustomerDomainRoute];
+
+const routeRenderMap: Record<APIRouteType, RouteTuple> = {
+  'organization-events': ['/organizations/:slug/events/', '/events/'],
```

Comment:
We also have slugs on team & project. Should we have prefixes to disambiguate what kind of slug is being interpolated?